README.md

```diff
@@ -2,31 +2,26 @@
 
 [![Semaphore Build Status Widget]][Semaphore Build Status] [![Travis Build Status Widget]][Travis Build Status] [![Coverage Status Widget]][Coverage Status] [![GoDoc Widget]][GoDoc] [![GoReportCard Widget]][GoReportCardResult] [![Slack Widget]][Slack] 
 
-## What is Kedge?
-
 ![logo](/docs/images/logo.png)
 
+----
+
 Kedge is a simple, easy and declarative way to define and deploy applications to Kubernetes by writing very concise application definitions.
 
 Key features and goals include:
 
+  - __An extension of Kubernetes:__ Use pre-existing definitions such as Pod or Container within your artifact file.
   - __Declarative:__ Declarative definitions specifying developer's intent.
   - __Simplicity:__ Using a simple and concise specification that is easy to understand and define.
   - __Concise:__ Define just the necessary bits and Kedge will do the rest. Kedge will interprolate and pick the best defaults for your application to run on Kubernetes.
   - __Multi-container environments:__ Define your containers, services and applications in one simple file, or split them into multiple files.
   - __Familiar structure:__ Using a familiar YAML structure as Kubernetes, it's easy to pick-up and understand Kedge.
-  - __Built on top of Kubernetes Pod definition:__ Leverages Kuberenetes Pod definition (PodSpec) and avoids leaky abstractions.
-
-
-## Project status
-
-We are a very evolving project with high velocity, we have listed a [file reference specification](docs/file-reference.md) as well as document our RFC's and changes as [GitHub issues](https://github.com/kedgeproject/kedge/issues).
 
-Check out our [roadmap](ROADMAP.md) as we push towards a __0.4.0__ release.
+----
 
-## Using Kedge
+## To start using Kedge
 
-### Installing
+### Install
```

Comment:
Can we link to installation.md somewhere in this section?
It includes some important information like latest builds, @pradeepto this information in readme

Reply:
I'll update it. Thanks!

Reply:
I put the latest builds in a separate installation document, but we could move this back to the README.md if you wish.

Reply:
👍 I'm fine with just linking it.